src/Storages/Distributed/DirectoryMonitor.cpp

```diff
@@ -136,16 +143,15 @@ void StorageDistributedDirectoryMonitor::shutdownAndDropAllData()
 
 void StorageDistributedDirectoryMonitor::run()
 {
-    setThreadName("DistrDirMonitor");
+    setThreadName(makeThreadName(path).c_str());
```

Comment:
Sometimes batches can be huge (especially if one will use big enough `distributed_directory_monitor_sleep_time_ms`), and in this case it is better to change the thread name

Reply:
So BackgroundSchedulePool does not the thread name, so dropping this code (and this will be resolved if separate pool will be used anyway)